index.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./home.css">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/2e9eda9b73.js" crossorigin="anonymous"></script>
+    <title>Mellisa Live Album Launch</title>
+</head>
+  <body>
+    <header>
+    <nav id="nav">
```

Comment:
- [ ] [Optional] The top bar has too much padding, kindly reduce the padding on the top bar and remove the small margin at the top

|Your design|CC design|
|---|---|
|![Screenshot_20220519_214810](https://user-images.githubusercontent.com/25227022/169402254-72f1e83c-6f9b-4a65-a051-cfc021168fbe.png)|![Screenshot_20220519_215247](https://user-images.githubusercontent.com/25227022/169402584-7fafdc15-60ee-4859-aa33-bbeb308da9c2.png)|

Reply:
Yes true. I removed padding and margin-top. I also changed the color 😄 something I noticed after you mentioned. Thanks @Ismailco